docs/changelog/3_x.rst

```diff
@@ -0,0 +1,726 @@
+====
+v3.0
+====
+
+:edb-alt-title: EdgeDB v3
```

Comment:
Add a warning saying that the latest stable is 2.x and 3.0 is expected to be released soon